src/actions/tests/__snapshots__/pending-breakpoints.spec.js.snap

```diff
@@ -123,3 +123,28 @@ Object {
   },
 }
 `;
+
+exports[`when adding breakpoints adding and deleting breakpoints add a corresponding pendingBreakpoint for each addition 3`] = `
+Object {
+  "astLocation": Object {
+    "name": undefined,
+    "offset": Object {
+      "line": 5,
+      "sourceId": "foo2",
+      "sourceUrl": "http://localhost:8000/examples/foo2",
+    },
+  },
+  "condition": null,
+  "disabled": false,
+  "generatedLocation": Object {
+    "column": undefined,
+    "line": 5,
+    "sourceUrl": "http://localhost:8000/examples/foo2",
+  },
+  "location": Object {
+    "column": undefined,
+    "line": 5,
+    "sourceUrl": "http://localhost:8000/examples/foo2",
+  },
+}
+`;
```

Comment:
how'd this change :)

Reply:
AGH!  Thank you!  I couldn't figure out why this wouldn't compile correctly, as I had only changed documentation.  I was trying to push this to my fork and it ran all the tests and it seemed to have changed this.